upload_log_files.py

```diff
@@ -0,0 +1,139 @@
+import argparse
+import os
+import re
+
+from core_data_modules.logging import Logger
+from storage.google_cloud import google_cloud_utils
+
+from src.lib import PipelineConfiguration
+
+log = Logger(__name__)
+
+
+def get_file_paths(dir_path):
+    # search for .gzip (data archive) and .profile (memory profile) files only because os.listdir(dir_path)
+    # returns all files in the directory
+    log_files_list = [file for file in os.listdir(dir_path) if file.endswith((".gzip", ".profile"))]
+    log_file_paths = [os.path.join(dir_path, basename) for basename in log_files_list]
+
+    return log_file_paths
+
+def fetch_latest_modified_file_path(dir_path):
```

Comment:
```suggestion
def get_latest_modified_file_path(dir_path):
```
Again, I think fetch implies retrieval over a network, which you're not doing here

Reply:
updated thanks